src/main/resources/twilio-go/api.mustache

```diff
@@ -38,26 +38,29 @@ type {{{nickname}}}Params struct {
 }
 {{/hasOptionalParams}}
 
-/*
-* {{operationId}}{{#summary}} {{{.}}}{{/summary}}{{^summary}} Method for {{operationId}}{{/summary}}
+// {{operationId}}{{#summary}} {{{.}}}{{/summary}}{{^summary}} Method for {{operationId}}{{/summary}}
+//
 {{#notes}}
-* {{notes}}
+// {{notes}}
+//
 {{/notes}}
 {{#allParams}}
 {{#required}}
-* @param {{paramName}}{{#description}} {{{.}}}{{/description}}
+// param: {{paramName}}{{#description}} {{{.}}}{{/description}}
+//
```

Comment:
Is this empty line needed?

Reply:
Yes, it functions as a line break for godocs generation. [This is an example](https://pkg.go.dev/github.com/twilio/twilio-go@v0.6.0/twilio/rest/accounts/v1#DefaultApiService.CreateCredentialAws) of formatting without the line breaks.

Reply:
[Here is a pointer](https://stackoverflow.com/a/51641739) to the [blog post](https://blog.golang.org/godoc) that documents this functionality.

Reply:
Hmm I don't like that go does that :unamused: